designs/developer/index.md

```diff
@@ -15,13 +15,30 @@ Our design documentation covers the entire developer experience in the OpenShift
 -- coming soon --
 
 ## 4.5 Developer Experience
-The following section documents the 4.5 OpenShift Console Developer experience, not just what's new in 4.5
+The following section document the 4.5 OpenShift Console Developer experience.
 
 ### Main Pages
 - [+Add](https://openshift.github.io/openshift-origin-design/designs/developer/add-45/)
+- [Builds](https://openshift.github.io/openshift-origin-design/designs/developer/builds/)
+- [Helm](https://openshift.github.io/openshift-origin-design/designs/developer/helm45/)
```

Comment:
should we keep the directory with a hyphen to be consistent?

Reply:
actually, the file is in a helm-45 directory, so this link should be changed!

Reply:
Yup!